packages/fixie/src/main.ts

```diff
@@ -102,76 +135,103 @@ registerServeCommand(program);
 program
   .command('user')
   .description('Get information on the current user')
-  .action(async () => {
-    const client = await AuthenticateOrLogIn({ apiUrl: program.opts().url });
-    const result = await client.userInfo();
-    showResult(result, program.opts().raw);
-  });
+  .action(
+    catchErrors(async () => {
```

Comment:
Hmm, commander doesn't have a way to set this at a command level, so we don't need to manually wrap anything? Bummer.

Reply:
Yup